mappings/net/minecraft/util/collection/WeightingUtil.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_6011 net/minecraft/util/collection/WeightingUtil
```

Comment:
In fact this class can just be named `Weighting`, like `Dismounting` etc.

Reply:
Sure, I guess.